src/main/java/com/google/opensesame/github/GitHubGetter.java

```diff
@@ -54,4 +54,24 @@ public static GitHub buildGitHubInterface(String clientId, String clientSecret)
       return GitHub.connectAnonymously();
     }
   }
+
+  /**
+   * Sets the GitHub API interface to the provided instance so that all of the following calls to
+   * getGitHub() will return this instance. This is intended to be used for mocking the GitHub API
+   * interface while testing and should not be elsewhere.
+   *
+   * @param mockGitHub
+   */
+  public static void mockGitHub(GitHub mockGitHub) {
```

Comment:
call this `setGitHubForTests` because that is more descriptive of what this function is doing. Otherwise when I look at testing code and see `mockGitHub` I'll be confused since I'd expect a function called `mockGitHub` to actually create the mock.

Reply:
Solid point here. Thanks, will change.